typings/node-forge.d.ts

```diff
@@ -0,0 +1,25 @@
+declare module 'node-forge' {
```

Comment:
Didn't https://www.npmjs.com/package/@types/node-forge work?

Reply:
It isn't complete. I can fork it and contribute. Just feeling some time pressure. :P

Reply:
That’s OK then. We can fork and contribute in the future, roll our own for now.

DefinitelyTypes PRs get automatically merged after a week of no one asks for changes, which makes iteration possible but is too slow for us right now